validation/standard/standard-config-params-sepolia.toml

```diff
@@ -10,7 +10,7 @@ maximum_base_fee = "340_282_366_920_938_463_463_374_607_431_768_211_455"
 [l2_output_oracle]
 submission_interval = [1, 86400]
 l2_block_time = [2, 2]
-challenge_period_seconds = [12, 12] # i.e. finalization period
+challenge_period_seconds = [12, 604800] # i.e. finalization period
```

Comment:
On mainnet, the bounds are [604800,604800], i.e. we require this to be exactly one week. On testnets, we can allow shorter challenge periods.